blueprints/module-unification-addon/files/.travis.yml

```diff
@@ -22,7 +22,7 @@ env:
   matrix:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
-    - EMBER_TRY_SCENARIO=ember-lts-2.18
+      EMBER_TRY_SCENARIO=ember-lts-2.18
```

Comment:
This change isn't correct. This one wasn't using stages yet, I think removing this `-` results in invalid yaml syntax.